ci/tasks/start-rds-instance.yml

```diff
@@ -0,0 +1,40 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: govukpay/concourse-runner
+params:
+  RDS_INSTANCE_NAME:
+  AWS_DEFAULT_REGION: eu-west-1
+run:
+  path: /bin/bash
+  args:
+    - -euo
+    - pipefail
+    - -c
+    - |
+      CURRENT_STATE=$(
+        aws rds describe-db-instances \
+          --db-instance-identifier "${RDS_INSTANCE_NAME}" \
+          --query "DBInstances[0].DBInstanceStatus" \
+          --output text
+      )
+
+      if [ "${CURRENT_STATE}" == "available" ]; then
+        echo "RDS instance ${RDS_INSTANCE_NAME} is already available"
+        exit 0
+      elif [ "${CURRENT_STATE}" == "starting" ]; then
+        echo "RDS instance ${RDS_INSTANCE_NAME} is already starting up"
+      elif [ "${CURRENT_STATE}" != "stopped" ]; then
+        echo "RDS instance ${RDS_INSTANCE_NAME} is not in a state that can be started." \
+          "It's currently in ${CURRENT_STATE}, manual intervention will be required to resolve this."
+
+        exit 1
+      else
+        echo "Starting RDS instance ${RDS_INSTANCE_NAME}"
+        aws rds start-db-instance --db-instance-identifier "${RDS_INSTANCE_NAME}" >> /dev/null
+      fi
+
+      echo "Waiting for RDS instance ${RDS_INSTANCE_NAME} to be available"
+      aws rds wait db-instance-available --db-instance-identifier "${RDS_INSTANCE_NAME}"
```

Comment:
Might it be useful to have another line after this to say "RDS instance is now available"?

Reply:
Ah yeah good plan

Reply:
Done :)